ams/api/src/main/java/com/netease/arctic/ams/api/properties/CatalogMetaProperties.java

```diff
@@ -25,17 +25,25 @@ public class CatalogMetaProperties {
   public static final String STORAGE_CONFIGS_KEY_HDFS_SITE = "hadoop.hdfs.site";
   public static final String STORAGE_CONFIGS_KEY_CORE_SITE = "hadoop.core.site";
   public static final String STORAGE_CONFIGS_KEY_HIVE_SITE = "hive.site";
+  public static final String STORAGE_CONFIGS_KEY_REGION = "storage.s3.region";
+  public static final String STORAGE_CONFIGS_KEY_ENDPOINT = "storage.s3.endpoint";
 
-  public static final String STORAGE_CONFIGS_VALUE_TYPE_HDFS = "hdfs";
+  public static final String STORAGE_CONFIGS_VALUE_TYPE_HDFS_LEGACY = "hdfs";
+  public static final String STORAGE_CONFIGS_VALUE_TYPE_HADOOP = "Hadoop";
+  public static final String STORAGE_CONFIGS_VALUE_TYPE_S3 = "S3";
```

Comment:
Why abandon `hdfs` and use `Hadoop` instead?

Reply:
This is a question of choosing `Hadoop` or `hdfs` as the storage type, as discussed in #2007. 

I think `Hadoop` is more suitable because it not only supports `hdfs`  but also supports object storage, like with `S3A`.